### PR TITLE
Add breadcrumbs, change physical  back button behaviour

### LINKED ISSUE #8: Better support of android physicals buttons
better control of backspace button
display menu on menu button
display search on search button

### PR DESCRIPTION
**Pull request by jacquarg**
- Fixes some errors messages translations
- Physical back close menu, and walk back through navigation history (and not files tree anymore) fix #8  .
- Add sliding and clickable breadcrumbs in top bar.
- Update actionbar app icon with svg version with paddings more _android compliants_
- Fix slide to open and close menu over files items.

Testable apk here : https://guillaumejacquart.cozycloud.cc/public/files/files/b84d8e2861c1f8cfe0b8a47dfa493294/attach/cozy_files-debug-20150220-1.apk

Needs brunch before merge.

### CODE REVIEW

**Comment by jacquarg** [comment]
Ready to review (I have no sufficient rights on this repo to chage tags).

**Comment by poupotte** [comment]
It's fonctionaly tested :)

File: www-src/app/locales/fr.coffee [hunk 1]
```diff
@@ -48,7 +49,7 @@ module.exports =
         La sauvegarde n'aura pas lieu car vous n'avez pas assez de batterie.
     """
     "no wifi"  : """
-        La sauvegarde n'aura pas lieu car vous n'êtes pas en wifi.
```
Comment (aenario) [inline www-src/app/locales/fr.coffee]: We were supposed to backup only on wifi to save bandwith, was that removed ?

**Comment by aenario** [comment]
Nothing to say code-wise. 
Did we remove the sync on wifi only ? Otherwise, the new error message is surprising if the user has 3G but no wifi.

**Comment by jacquarg** [comment]
Ok, I may change it too quickly. I will look deeper about wifi / 3G , and what's the true behavior of the 'backup in wifi only' option.

**Comment by aenario** [comment]
Thanks @jacquarg  !
@poupotte , please publish on the Play Store when you're done testing.

**Comment by frankrousseau** [comment]
@poupotte and update the ACK file on files.cozycloud.cc .

**Pull request merged**